regparser/tree/xml_parser/note_processor.py

```diff
@@ -5,6 +5,7 @@
 
 
 class IgnoreNotesHeader(paragraph_processor.BaseMatcher):
+    """We don't want to include "Note:" and "Notes:" headers"""
```

Comment:
Is this too generic? Am I being paranoid about running into something like a header starting “Noted in our consideration of this proposal were the following…”? If I'm not being too paranoid, something like `return xml.tag == 'HD' and xml.text.lower().split(":", 1)[0] in ("note", "notes")` might be better.